label_studio/frontend/src/pages/CreateProject/Import/Import.js

```diff
@@ -13,6 +13,24 @@ function flatten(nested) {
   return [].concat(...nested);
 }
 
+const supportedFormats = {
+  text: ['txt'],
+  audio: ['wav', 'aiff', 'mp3', 'au', 'flac', 'm4a', 'ogg'],
```

Comment:
```suggestion
  audio: ['wav', 'mp3', 'flac', 'm4a', 'ogg'],
```
These are the only supported audio formats of webaudio. We can just remove the `au` and `aiff`